### PR TITLE
update change logs with lease put improvements

### PR DESCRIPTION
**Pull request by tjungblu**
Let me know if that wording works for you.

for #15993

### CODE REVIEW

**Review by jmhbnz — APPROVED** [review]
Wording makes sense to me, thanks @tjungblu.

Edit: Just need the linked pr's to merge before we hit merge on this one.

File: CHANGELOG/CHANGELOG-3.6.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0).
 - Add [`etcd --experimental-snapshot-catch-up-entries`](https://github.com/etcd-io/etcd/pull/15033) flag to configure number of entries for a slow follower to catch up after compacting the the raft storage entries and defaults to 5k. 
 - Decreased [`--snapshot-count` default value from 100,000 to 10,000](https://github.com/etcd-io/etcd/pull/15408)
 - Add [`etcd --tls-min-version --tls-max-version`](https://github.com/etcd-io/etcd/pull/15156) to enable support for TLS 1.3.
+- Improve [Lease put performance with disabled auth](https://github.com/etcd-io/etcd/pull/16005)
```
Comment (serathius) [inline CHANGELOG/CHANGELOG-3.6.md:78]: We don't put changes to v3.6 that we intend to backport.

Comment (tjungblu) [reply CHANGELOG/CHANGELOG-3.6.md]: removed

**Review by serathius — APPROVED** [review]

**Review by fuweid — APPROVED** [review]
LGTM

File: CHANGELOG/CHANGELOG-3.4.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 
 ### etcd server
 - Fix [corruption check may get a `ErrCompacted` error when server has just been compacted](https://github.com/etcd-io/etcd/pull/16047)
+- Improve [Lease put performance with disabled auth](https://github.com/etcd-io/etcd/pull/16020)
```
Comment (ahrtr) [inline CHANGELOG/CHANGELOG-3.4.md:11]:
```suggestion
- Improve [Lease put performance for the case that auth is disabled or the user is admin](https://github.com/etcd-io/etcd/pull/16020)
```

File: CHANGELOG/CHANGELOG-3.5.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ### etcd server
 - Fix [corruption check may get a `ErrCompacted` error when server has just been compacted](https://github.com/etcd-io/etcd/pull/16048)
+- Improve [Lease put performance with disabled auth](https://github.com/etcd-io/etcd/pull/16019)
```
Comment (ahrtr) [inline CHANGELOG/CHANGELOG-3.5.md:11]: same as above

**Comment by tjungblu** [comment]
updated the wording, thanks for the review @ahrtr

**Review by ahrtr — APPROVED** [review]
LGTM

thx @tjungblu

**Pull request merged**